public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -1339,16 +1340,29 @@ export class DashboardModel implements TimeModel {
     return this.originalDashboard;
   }
 
-  hasAngularPlugins(): boolean {
-    return this.panels.some((panel) => {
+  async hasAngularPlugins(): Promise<boolean> {
+    for (const panel of this.panels) {
+      let plugin;
```

Comment:
nit: this `plugin` variable should be a constant in line 1348

Reply:
Fixed!